libs/base/ui/structure/collapsible/collapsible.component.ts

```diff
@@ -67,8 +77,26 @@ export class CollapsibleComponent
     }
   }
 
-  protected onClick(): void {
-    this.isManuallyOpened = true;
+  protected syncState(store = false): void {
+    if (!this.persistedStateKey) return;
+
+    const uiStorageKey = 'ui';
```

Comment:
this key is to generic for a low level framework component

Reply:
`oryx-ui`?

Reply:
what about `oryx-ui.collapsible` as a key? instead of serializing/deserializing one big object for every possible future UI component (may grow)?

Reply:
changed to `oryx-collapsible`